solutions/USACO feb17-circlecross.mdx

```diff
@@ -2,223 +2,76 @@
 id: usaco-712
 source: USACO Bronze 2017 February
 title: Why Did the Cow Cross the Road II
-author: Benjamin Qi
+author: Benjamin Qi, Danh Ta Chi Thanh
 ---
 
 [Official Analysis](http://www.usaco.org/current/data/sol_circlecross_bronze_feb17.html)
 
+Let the first occurrence of a letter be the cow's entry and the last occurrence her exit. Then, we count the number of pairs $(cow_1, cow_2)$ such that $cow_1$ enters before $cow_2$ and leaves somewhere between $cow_2$'s entry and exit. 
+
+In other words, we define $start_x$, $end_x$, as the entry and exit points of a $cow_x$, where $start_x$ is the index of the first occurrence of $cow_x$ and $end_x$ is the index of the last occurrence of $cow_x$. Thus, we count the number of pairs of cows $(x, y)$ such that $start_x < start_y < end_x < end_y$.
```

Comment:
> In other words, we define $start_x$, $end_x$, as the entry and exit points of a $cow_x$, where $start_x$ is the index of the first occurrence of $cow_x$ and $end_x$ is the index of the last occurrence of $cow_x$.

Just say

> Define $start_x$ to be the index of the first occurrence of $cow_x$ and $end_x$ to be the index of the last occurrence of $cow_x$.

Reply:
Yeah the explanation becomes more concise after doing this

Reply:
Is there anything that needs to be changed?
Could you list them all? I will commit later because it is 11:34 P.M. in my time zone so I'm gonna take a sleep.

Reply:
Thanks for being on top of the comments and dealing with them so promptly! It's a lot, but I'd rather have too much quality assurance than too little :)

Reply:
Sleep is always important

Reply:
> Is there anything that needs to be changed?

That's all I have for now :)